integration-cli/docker_cli_events_test.go

```diff
@@ -411,3 +412,121 @@ func checkEvents(t *testing.T, events []string) {
 	}
 
 }
+
+func TestEventsStreaming(t *testing.T) {
+	testRequires(t, SameHostDaemon)
+
+	startEpoch := time.Now().Add(-1 * time.Second)
```

Comment:
Use `since := daemonTime(t).Unix()`, this should eliminate need for `testRequires(t, SameHostDaemon)`